email.php

```diff
@@ -0,0 +1,25 @@
+<?php
+    if(isset($_POST["submit"])){
+        $email=$_POST["email"];
+        $msg="";
+        if(strpos($email, "@")!=false){
+        for($i=0;$i<strlen($email);$i++){
+            if(($email[$i]>='a' && $email[$i]<='z')||
```

Comment:
If we are trying to do email validation here , approach can be better
1. We can use php `FILTER_VALIDATE_EMAIL`  filter
2. We can use regex
3. We can use some email validtion package/service